homeassistant/components/denonavr/strings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Denon AVR Network Receivers",
+        "description": "Connect to your receiver, if the IP address is not set, auto-discovery is used",
+        "data": {
+          "host": "IP address"
+        }
+      },
+      "settings": {
+        "title": "Denon AVR Network Receivers",
+        "description": "Specify optional settings",
+        "data": {
+          "timeout": "Timeout in seconds",
+          "show_all_sources": "Show all sources",
+          "zone2": "Set up Zone 2",
+          "zone3": "Set up Zone 3"
+        }
+      },
+      "select": {
+        "title": "Select the receiver that you wish to connect",
+        "description": "Run the setup again if you want to connect aditional receivers",
```

Comment:
`aditional` is misspelled. Should be `additional`

Reply:
thanks, I just fixed it